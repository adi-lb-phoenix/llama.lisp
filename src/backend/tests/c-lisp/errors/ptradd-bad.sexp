(c-lisp
    (define ((main int))
        (declare arr (ptr int))
        (set arr (alloc int 20))
        (call print (ptradd arr 4 3))))
