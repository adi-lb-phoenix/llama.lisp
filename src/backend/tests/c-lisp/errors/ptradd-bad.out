Error in statement: ['call', 'print', ['ptradd', 'arr', 4, 3]]
CodegenError: Bad ptradd expression: ['ptradd', 'arr', 4, 3]
